server/prpc/server.go

```diff
@@ -111,8 +111,7 @@ func (s *Server) authenticate() router.Middleware {
 	if a == nil {
 		a = GetDefaultAuth()
 		if a == nil {
-			panic("prpc: no custom Authenticator was provided and default authenticator was not registered. " +
```

Comment:
Yeah sorry, I should have said, `[]auth.Authenticator{}`.

Perhaps the better option here is to declare a package global called `NoAuthenticator == []auth.Authenticator{}` and have the user supply that if they want no authenticator.